sendwp/init.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+include_once CFCORE_PATH . 'sendwp/handler.php';
+
+function caldera_forms_admin_enqueue_sendwp_installer() {
```

Comment:
Could the the function be commented too ?

Reply:
Added a dockblock comment.